// @atlaskit/section-message/dist/cjs/theme.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.baseAppearanceObj = void 0;
var tslib_1 = require("tslib");
var check_circle_1 = tslib_1.__importDefault(require("@atlaskit/icon/glyph/check-circle"));
var error_1 = tslib_1.__importDefault(require("@atlaskit/icon/glyph/error"));
var info_1 = tslib_1.__importDefault(require("@atlaskit/icon/glyph/info"));
var question_circle_1 = tslib_1.__importDefault(require("@atlaskit/icon/glyph/question-circle"));
var warning_1 = tslib_1.__importDefault(require("@atlaskit/icon/glyph/warning"));
var colors = tslib_1.__importStar(require("@atlaskit/theme/colors"));
exports.baseAppearanceObj = {
    info: {
        backgroundColor: colors.B50,
        Icon: info_1.default,
        primaryIconColor: colors.B500,
    },
    warning: {
        backgroundColor: colors.Y50,
        Icon: warning_1.default,
        primaryIconColor: colors.Y500,
    },
    error: {
        backgroundColor: colors.R50,
        Icon: error_1.default,
        primaryIconColor: colors.R500,
    },
    confirmation: {
        backgroundColor: colors.G50,
        Icon: check_circle_1.default,
        primaryIconColor: colors.G500,
    },
    change: {
        backgroundColor: colors.P50,
        Icon: question_circle_1.default,
        primaryIconColor: colors.P500,
    },
};
